content/docs/command-reference/studio/index.md

```diff
@@ -0,0 +1,37 @@
+# studio
+
+Set of commands to authorize DVC with Studio and save the token to global [DVC
```

Comment:
let's review here in the code carefully if it's authorize or authenticate.

E.g. here https://cli.github.com/manual/gh_auth_login and here https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli it is authenticate.

We should be careful about this terminology.